open-api/rest-catalog-open-api.yaml

```
@@ -1873,6 +1886,15 @@ components:
         type: integer
         minimum: 1
 
+    etag:
+      name: ETag
+      in: header
+      description: Identifies a certain state of the table metadata.
```

Is `version` more accurate here?
```suggestion
      description: Identifies a unique version of the table metadata.
```

---

Sure, suggestion applied